pkgs/jnigen/test/simple_package_test/runtime_test_registrant.dart

```diff
@@ -36,7 +36,8 @@ void registerTests(String groupName, TestRunnerCallback test) {
       expect(Example.OFF, equals(0));
       expect(Example.PI, closeTo(pi, fpDelta));
       expect(Example.SEMICOLON, equals(';'.codeUnitAt(0)));
-      expect(Example.SEMICOLON_STRING, equals(';'));
+      expect(Example.SEMICOLON_STRING.toDartString(releaseOriginal: true),
+          equals(';'));
```

Comment:
Add an emoji as well in pkgs/jnigen/test/simple_package_test/java/com/github/dart_lang/jnigen/simple_package/Example.java ?